blazor/datagrid/how-to/customize-filter-popup-using-css.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Customize filter popup using CSS in Blazor DataGrid | Syncfusion
+description: Learn here all about Customizing filter popup using CSS in Syncfusion Blazor DataGrid component and more.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Customize Filter Popup Using CSS
+
+You can set the height and width for each column in the filter popup dialog using the CSS style.
+
+In the following sample, we have enabled an Excel like filter by defining Type as `Excel`. You can set the height and width for each column in the popup filter based on the condition in the [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnActionBegin) event of the Grid.
```

Comment:
In the following sample, we have enabled an Excel-like filter by defining Type as `Excel`. You can set the height and width of each column in the pop-up filter based on the condition in the [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnActionBegin) event of the Grid.